.travis.yml

```diff
@@ -24,4 +24,6 @@ install:
 
 
 script:
- - python -m unittest discover
\ No newline at end of file
+ - #check that doc generation is possible
+  - generate_docs.sh
```

Comment:
I'm pretty sure you need to say `./generate_docs.sh` or `bash generate_docs.sh` here. Travis build for this PR is currently failing on this line.

Reply:
oops yeah, I had that fixed on my local version. should be ./